### PR TITLE
layout_catalog-demo

### PR DESCRIPTION
**Pull request by RSydorenko322**
- [DEMO LINK](https://RSydorenko322.github.io/layout_catalog/)
- [TEST REPORT LINK](https://RSydorenko322.github.io/layout_catalog/report/html_report/)

❗️ Copy this `Checklist` to the `Pull Request` description after links, and put `- [x]` before each point after you checked it.

- [x] All component follow BEM and use SCSS
- [x] repaeted sizes and special colors are put to variables
- [x] Grid is used for the columns
- [x] cards are shown in 1, 2, 3 or 4 columns based on screen resolution
- [x] All changes on `:hover` are smooth
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,365 @@
...
+        </div>
+
+        <div class="card__price card__price--spaced">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-sum">$2,199</div>
+        </div>
+
+        <a href="#"
+        class="card__button"
+        data-qa="card-hover">
```
Comment (lerastarynets) [inline src/index.html:110]: attrCheklist

**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Nice start! Let's fix the solution according to the checklist

File: src/index.html [hunk 2]
```diff
@@ -4,9 +4,358 @@
...
+          </div>
+        </div>
+
+        <div class="card__price card__price--spaced">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-sum">$2,199</div>
+        </div>
+
+        <a href="#" class="card__button"
+        data-qa="card-hover">
```
Comment (lerastarynets) [inline src/index.html:110]: attrCheklist

If a mentor points out a mistake, it means that you need to fix ALL occurrences of the mistake, not only in the place where the comment was left.

**Review by lerastarynets — APPROVED** [review]
Great job overall! Left one tiny suggestion for the improvement

File: src/index.html (hunk 2 above)
Comment (RSydorenko322) [reply src/index.html:110]: I apologize , really missed that by accident

**Pull request closed without merging**